README.md

```diff
@@ -71,195 +70,190 @@ are available. Please click on the links below for more informations and picture
-bluetooth headset is connected LEDs turn to blue. Please note: due to memory-restrictions it's not
-possible to run bluetooth in parallel with WiFi. This finally means that you cannot stream webradio
-via bluetooth or access webgui while this mode is enabled.
+immediately. Activated Bluetooth is indicated by four slow rotating _blue-violet_ LEDs. After the
+Bluetooth headset is connected LEDs turn to blue.
 
-## Port-expander
+## Port Expander
 
-There might be situations where you run out of GPIOs. To address this, port-expander
-[PCA9555](https://www.nxp.com/docs/en/data-sheet/PCA9555.pdf) can be used to extend number of
-input-channels (output-mode is only supported in special cases). PCA9555 provides 2 ports with 8
+There might be situations where you run out of GPIOs. To address this, a port expander
+[PCA9555](https://www.nxp.com/docs/en/data-sheet/PCA9555.pdf) can be used to extend the number of
+input channels (output mode is only supported in special cases). PCA9555 provides 2 ports with 8
 channels each - so 16 channels in total. To activate PCA9555 you need to enable
 `PORT_EXPANDER_ENABLE`. Like GPIOs in your develboard-specific settings-file, you can assign
-numbers. Range is 100->115 where 100: port 0 channel 0 -> 107: port 0 channel 7; 108: port 1 channel
-0 -> 115: port 1 channel 7. Via `expanderI2cAddress` port-expander's I2C-address can be changed.
-It's `0x20` if all A0, A1, A2 are wired to GND.
+numbers. Range is `100` (port 0 channel 0) -> `115` (port 1 channel 7). Via `expanderI2cAddress` the
+port expander's I2C-address can be changed. It's `0x20` if pins `A0`, `A1`, `A2` are wired to GND.
 
 ## After ESPuino is connected to your WiFi
 
-After making ESPuino part of your LAN/WiFi, the 'regular' webgui is available at the IP assigned by
+After making ESPuino part of your LAN/WiFi, the 'regular' web GUI is available at the IP assigned by
 your router (or the configured hostname). Using this GUI you can:
 
 - configure WiFi
-- make bindings between RFID-tag, file/directory/URL and playMode
-- make bindings between RFID-tag and a modification-type
+- make bindings between RFID tag, file/directory/URL and playback mode
+- make bindings between RFID tag and a modification type
 - configure MQTT (if enabled)
 - configure FTP (if enabled)
-- configure initial volume, maximum volume (speaker / headphone), brightness of Neopixel (nightmode
-  / default) and inactivity-time
+- configure initial volume, maximum volume (speaker / headphone), brightness of Neopixel (night mode
+  / default) and inactivity time
 - view logs / status / current track
 - control player
-- run modifications (like modification-card)
-- upload audiofiles (called webtransfer)
-- do OTA-updates (ESP32s with 16 MB of flash-memory only)
+- run modifications (like modification card)
+- upload audiofiles (called web transfer)
+- do OTA updates (ESP32 with 16 MB of flash memory only)
```

Comment:
Just a note: I've been running the 8MB partition scheme with OTA capability for quite some time. Maybe it can be added as a side note here.

Reply:
As this PR is about formal changes only, I'll leave this one for a future PR as it is about the actual content.